blakserv/webhook.c

```diff
@@ -0,0 +1,222 @@
+// Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+// All rights reserved.
+//
+// This software is distributed under a license that is described in
+// the LICENSE file that accompanies it.
+//
+// Meridian is a registered trademark.
+/*
+* webhook.c
+*
+
+  Cross-platform webhook message delivery via named pipes (Windows) or FIFOs (Linux/macOS).
+  Keeps pipes open for performance instead of opening/closing on every message.
+
+*/
+
+#include "blakserv.h"
+#include "webhook.h"
+#include <time.h>
+
+#ifdef BLAK_PLATFORM_WINDOWS
+#include <windows.h>
+#else
+#include <fcntl.h>
+#include <unistd.h>
+#endif
+
+// Configuration constants
+static const int MAX_WEBHOOK_PIPES_WINDOWS = 10;
+static const int MAX_WEBHOOK_PIPES_UNIX = 10;
```

Comment:
Do you really need platform-specific constants?  Seems like one would be enough.